src/OpenLoco/src/Vehicles/OrderManager.cpp

```diff
@@ -280,6 +280,45 @@ namespace OpenLoco::Vehicles::OrderManager
         }
     }
 
+    uint16_t reverseVehicleOrderTable(uint16_t tableOffset, uint16_t orderOfInterest)
+    {
+        // Retrieve list of raw orders
+        std::vector<uint64_t> rawOrders{};
```

Comment:
We should probably have a note somewhere in the order manager that you can't use std:: algorithms for a lot of this due to the variable length of elements. Then hopefully when we stop using variable length we can simplify a huge amount of this.

Reply:
Completely agreed. I'll add the note in a few places. I'm thinking something among the lines of:
```
// Currently the use of std:: algorithms is not feasible due to variable order lengths
// TODO: simplify this after the changing the data structure for the order table
```